### DISCUSSION TITLE
FastAPI with Pydantic v2 and FastAPI-JWT-Auth

### DISCUSSION THREAD

**Discussion by Kumbukani-ck** [fastapi/fastapi#10718] [Questions] [answered]
Pydantic v2 isn't working with FastAPI-JWT-Auth i'm always getting the Type Error highlighting that @validators can not be applied on schemas of  json-to-python.

So is there any other alternative other than downgrading to version 1?

**Comment by jgould22** [comment, marked as answer]
The solution is to get the maintainer to update their package, fork it and update it yourself or replace it.

You can see some people talking about that here https://github.com/IndominusByte/fastapi-jwt-auth/issues/98
